rustc-echo-wrapper

```diff
@@ -0,0 +1,5 @@
+#! /bin/bash
```

Comment:
I think instead of checking in a file, we could just have the tests call a setup function that will use `fs::write` to create the file in `paths::root()`.   It's a bit unfortunate not supporting windows, but maybe not worth the hassle.